internal/espresso/config.go

```diff
@@ -24,7 +24,8 @@ var (
 // Project represents the espresso project configuration.
 type Project struct {
 	config.TypeDef `yaml:",inline" mapstructure:",squash"`
-	Defaults       config.Defaults        `yaml:"defaults" json:"defaults"`
+	Defaults       config.Defaults `yaml:"defaults" json:"defaults"`
+	ShowConsoleLog bool
```

Comment:
```suggestion
	ShowConsoleLog bool `yaml:"showConsoleLog"`
```

Otherwise it's case is inconsistent with our other attributes. Can you also do that for the other frameworks, please (I know it's not related to this PR, strictly speaking, but might as well)?

Reply:
sure.